src/docbkx/chapter41-configuration-file.xml

```diff
@@ -513,7 +513,7 @@
 
 	<section id="configuration_file_task_runner">
 		<title>Task runner</title>
-		<para>The task runner defines how DataCleaner's will engine will
+		<para>The task runner defines how DataCleaner's engine will
```

Comment:
Nonono... It's a Will Engine... An engine running on pure will! ;)

(Or an engine running on guys named Will... I'm not entirely sure)